src/app/main/component/events/components/events-list/events-list.component.html

```diff
@@ -54,9 +54,9 @@
             </mat-option>
           </mat-select>
         </div>
-        <div class="dropdown">
+        <div class="dropdown" (click)="selectStatusesList.open()">
```

Comment:
Don't need to add` #selectStatusesList `- you can write already added #statusFilter -` statusFilter.open()`

Reply:
Thank you! Add changes!